be/test/vec/function/function_array_aggregation_test.cpp

```diff
@@ -0,0 +1,122 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <type_traits>
+
+#include "function_test_util.h"
+#include "vec/data_types/data_type_number.h"
+
+namespace doris {
+namespace vectorized {
+
+using IntDataSet = std::vector<std::pair<std::vector<int>, int>>;
+
+template <typename T, typename ReturnType = T>
+void check_function(const std::string& func_name, const IntDataSet data_set) {
+    InputTypeSet input_types = {TypeIndex::Array, ut_type::get_type_index<T>()};
+    DataSet converted_data_set;
+    for (const auto& row : data_set) {
+        Array array;
+        for (auto value : row.first) {
+            array.push_back(ut_type::convert_to<T>(value));
+        }
+        converted_data_set.emplace_back(std::make_pair<CellSet, Expect>(
+                {array}, ut_type::convert_to<ReturnType>(row.second)));
+    }
+    check_function<ReturnType>(func_name, input_types, converted_data_set);
+}
+
+TEST(FunctionArrayAggregationTest, TestArrayMin) {
```

Comment:
better to add  `V` as a prefix of vectorized test

Reply:
Added.